src/tree/index.ts

```diff
@@ -38,14 +40,27 @@ const ripOutPaths = (
     ? maxDepthLevel
     : findMaximumDepthLevel(responseBody);
 
+  /** Given a path, searches through old Tree and Builtin Tree for the path and returns if any comment is placed there
+   * @param {string} path - path of the core
+   * @returns {string} - default comment for the core
+   */
   const setComment = (path) => {
-    const found = oldTree
-      ? oldTree.find((item) => item.path == START_OF_PATH + path)
-      : null;
-    if (found) {
-      return START_OF_COMMENT + found.comment;
+    let builtinComment = "";
```

Comment:
This function is not easily readable, try extracting the logic into smaller functions. EX:

const setComment = () => {
 getBuiltInComment
 getOldComment
 getPreDefinedComment
}
See how by extracting the code into functions I've made it super clear what the function is doing that you can read it :) (This is a more advanced code quality standard usually done for backends but always good to practice it)